samples/visual_indicator/nodejs/README.md

```diff
@@ -0,0 +1,129 @@
+---
+page_type: sample
+description: Microsoft Teams meeting extensibility sample for iteracting with In-meeting notifications
+products:
+- office-teams
+- office
+- office-365
+languages:
+- nodejs
```

Comment:
I can't see this sample is linked to Root Readme.

Also, build and deployment pipeline is missing.